### ISSUE TITLE
conduit staying in cf apps after ctrl+c

### ISSUE THREAD

**Issue by leohemsted** [alphagov/paas-cf-conduit#26]
org `govuk-notify`, space `production`, conduit name `__conduit_79086__`

Timeline:

conduit created (`cf conduit notify-db -- psql`) at 10:23
used normally at 10:24:59
entered another query at 10:44:46 - no response
killed manually using ctrl+c at 10:48:
```
rdsbroker_**********=> select * from notifications where sent_by = 'mmg' limit 5;
^[[A^[[A^[[B^[[B^C
[leohemsted:~]$ Could not send cancel request: PQcancel() -- connect() failed: Connection refused
SSL SYSCALL error: EOF detected
The connection to the server was lost. Attempting reset: Failed.
!> \q

[leohemsted:~]$
```
10:49 conduit still visible in cf apps

**Comment by benvand** [comment]
+1

Would be good to get some documentation on how the plugin handles dropped connections, timeouts, cleanup of stale apps etc :+1: Otherwise nice work no the plugin. Super easy to install and use :100:.

**Comment by matthewford** [comment]
yup had to manually delete them

**Comment by matthewford** [comment]
duplicate of https://github.com/alphagov/paas-cf-conduit/issues/18

**Comment by bandesz** [comment]
This is now solved with the latest release (0.0.4)
